### PR TITLE
Igal/sno

### PR DESCRIPTION
**Pull request by tsorya**
Will add tests after review

### CODE REVIEW

File: src/installer/installer.go [hunk 1]
```diff
@@ -71,41 +70,52 @@ func (i *installer) InstallNode() error {
...
-		isBootstrap = true
+
+	if isBootstrap {
 		errs.Go(func() error {
 			return i.runBootstrap(ctx)
 		})
 		i.Config.Role = string(models.HostRoleMaster)
 	}
 
+	if err := i.installNode(isSingleNodeBootstrap); err != nil {
```
Comment (eranco74) [inline src/installer/installer.go:91]: Just don't call installNode if `isSingleNodeBootstrap == true`
```suggestion
	if isSingleNodeBootstrap {
            i.log.Infof("Bootstrap in single node mode, skipping installNode")
	} else {
	    if err := i.installNode(); err != nil {
```

File: src/installer/installer.go [hunk 2]
```diff
@@ -115,19 +125,47 @@ func (i *installer) InstallNode() error {
...
+	return nil
+}
+
+func (i *installer) finalizingInstallation(isBootstrap bool, isSingleNodeBootstrap bool) error {
+
+	stage := models.HostStageRebooting
+	operation := i.ops.Reboot
+	if isSingleNodeBootstrap {
+		stage = models.HostStageDone
+		operation = i.ops.Shutdown
```
Comment (eranco74) [inline src/installer/installer.go:158]: I think it's safer to have no-op operation in case of single node

Comment (tsorya) [reply src/installer/installer.go:158]: possible, thought it will be better

**Pull request closed without merging**